docroot/sites/classrooms.uiowa.edu/modules/classrooms_core/templates/field--node--field-room-seating-chart--room--default.html.twig

```diff
@@ -1,6 +1,6 @@
-{%- for item in items -%}
-  <div class="bg--gold bttn bttn--primary bttn--disabled bttn--light-font bttn--no-text bttn--small">
-    <i role="presentation" class="fa-file-pdf fas"></i>&nbsp;
-    <a  href="{{ item.content|render }}">Seating Chart</a>
-  </div>
-{%- endfor -%}
+{% embed '@uids_base/field/field.html.twig' %}
+  {% block field_icon %}
+    <span role="presentation" class="fa-user fas"></span>
+  {% endblock %}
+  {% set attributes = attributes.addClass('bg--gold element--padding__all--sm') %}
+{% endembed %}
```

Comment:
```suggestion
{% set attributes = attributes.addClass('bg--gold element--padding__all--sm') %}

{% block field_icon %}
    <span role="presentation" class="fa-user fas"></span>
{% endblock %}
```
Not tested, but I think you can do this instead.

Reply:
We still need to embed or include the field template.

Reply:
I'm going to test this. I don't think we do need to include the embed.

Reply:
It still requires `extends` but it can be done without `embed`.